client/store/reports/stats/resolvers.js

```diff
@@ -15,7 +15,7 @@ import { NAMESPACE } from 'store/constants';
 export default {
 	async getReportStats( state, endpoint, query ) {
 		const statEndpoints = [ 'orders', 'revenue', 'products' ];
-		let apiPath = endpoint;
+		let apiPath = endpoint + stringifyQuery( query );
```

Comment:
Ah nice change here - I obviously failed to consider other endpoints wanting to use a query object 🤦‍♂️